src/components/Template/Template.mdx

```diff
@@ -47,72 +76,271 @@ import {
+    </List>
+  }
+  imageProps={{
+    alt: "Template top region",
+    caption: "Top region",
+    src: "/template/templateTop.png",
+  }}
+/>
+}
 
-- [WebAim Skip Navigation Link](https://webaim.org/techniques/skipnav/)
-- [A11ymatters Skip Navigation Link](https://www.a11ymatters.com/pattern/skip-link/)
+### TemplateSidebar
+
+{
 
-## Template Sidebar Configuration
+<DocsWithImage
+  docs={
+    <List type="ul">
+      <li>
+        Positioned below <code>TemplateTop</code> and to either the left or
+        right of <code>TemplateMain</code>
+      </li>
+      <li>
+        Not recommended for use with <code>TemplateMainNarrow</code>
+      </li>
+      <li>
+        Spans a perecentage of the central content area based on the current
+        breakpoint
+      </li>
+      <li>Ideal for filtering elements and secondary menus</li>
+    </List>
+  }
+  imageProps={{
+    alt: "Template sidebar region",
+    caption: "Sidebar region",
+    src: "/template/templateSidebarLeft.png",
+  }}
+/>
+}
 
-In order to render a sidebar, the `Template` component needs to be aware of it.
-Pass the location of the sidebar to the `sidebar` prop as either "left" or "right".
+### TemplateMain
 
-For accessibility purposes, you should also structure the DOM logically. This means
-if you are using `sidebar="left"`, place the `TemplateSidebar` before `TemplateMain`.
-If you are using `sidebar="right"`, place the `TemplateSidebar` after.
+{
+
+<DocsWithImage
+  docs={
+    <List type="ul">
+      <li>
+        Positioned below <code>TemplateTop</code> and to either the left or
+        right of
+        <code>TemplateSidebar</code>, when it is rendered
+      </li>
+      <li>
+        May be used in conjunction with <code>TemplateSidebar</code>
+      </li>
+      <li>
+        Spans a perecentage of the central content area when the sidebar is
+        rendered
+      </li>
+      <li>
+        Spans the full width of the central content area when the sidebar is not
+        rendered
+      </li>
+      <li>
+        Content in this region should adhere to the{" "}
+        <Link href="/?path=/docs/style-guide-responsive-grid--docs">
+          responsive grid
+        </Link>{" "}
+        recommendations
+      </li>
+    </List>
+  }
+  imageProps={{
+    alt: "Template main region",
+    caption: "Main region",
+    src: "/template/templateMain.png",
+  }}
+/>
+}
+
+### TemplateMainNarrow
+
+{
+
+<DocsWithImage
+  docs={
+    <List type="ul">
+      <li>
+        Positioned below <code>TemplateTop</code> and to either the left or
+        right of
+        <code>TemplateSidebar</code>, when it is rendered
+      </li>
+      <li>
+        Should not be used in conjunction with <code>TemplateSidebar</code>
+      </li>
+      <li>Contained by the central content area</li>
+      <li>
+        Max width of <code>720px</code>
+      </li>
+      <li>Ideal for displaying content that is primarily text</li>
```

Comment:
I'd move this up since it's the main reason for having this component.

Reply:
The recommendations for all the other child components are at the end of the list, so I am wary to break the pattern just for this one. That said, I will update all of the lists to move the recommendations to the top.